src/allmydata/crypto/rsa.py

```diff
@@ -0,0 +1,156 @@
+from cryptography.exceptions import InvalidSignature
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.asymmetric import rsa, padding
+from cryptography.hazmat.primitives.serialization import load_der_private_key, load_der_public_key, \
+    Encoding, PrivateFormat, PublicFormat, NoEncryption
+
+from allmydata.crypto import BadSignature
+
+
+"""
+This is the value that was used by `pycryptopp`, and we must continue to use it for
+both backwards compatibility and interoperability.
+
+The docs for `cryptography` suggest to use the constant defined at
+`cryptography.hazmat.primitives.asymmetric.padding.PSS.MAX_LENGTH`, but this causes old
+signatures to fail to validate.
+"""
+RSA_PSS_SALT_LENGTH = 32
```

Comment:
This should be a comment, not a string.

Reply:
Doesn't a string like this get rendered into the docs by Sphinx?

Reply:
Oof.  Does it?  I wasn't aware.  If so, okay, I guess Sphinx is our doc system, we have to do what it wants.  I would have guessed that the way to provide user-facing documentation for a module attribute is to put `:var ...:` into the module docstring.

*Do* we want to provide this as user-facing documentation?  And, hm, this would presumably end up in some kind of API-ish documentation rendered by Sphinx ... but I don't think Tahoe-LAFS generates any API documentation (since it doesn't really have a public Python API...).

Reply:
I just turned it into a comment ;) since, yeah, we don't have "a python API" anyway.